js/models/Settings.js

```diff
@@ -26,10 +30,40 @@ export default class extends BaseModel {
   nested() {
     return {
       shippingAddresses: ShippingAddresses,
-      smtpSettings: SMTPSettings,
+      appearanceSettings: AppearanceSettings,
```

Comment:
As we discussed, these are local settings for the localSettings model and I don't see a need for them to be nested.